pkgs/development/python-modules/deeplabcut/default.nix

```diff
@@ -0,0 +1,126 @@
+{ lib, buildPythonPackage, fetchPypi
+, python3
+, fetchzip
+, tensorflow
+, setuptools_scm
+, gzip
+, gnutar
+, certifi
+, chardet
+, click
+, easydict
+, h5py
+, imageio
+, imgaug
+, ipython
+, matplotlib
+, moviepy
+, numpy
+, opencv3
+, pandas
+, patsy
+, pillow
+, python
+, pyyaml
+, requests
+, ruamel_yaml
+, setuptools
+, scikitimage
+, scikitlearn
+, scipy
+, six
+, statsmodels
+, tables
+, tensorpack
+, tqdm
+, tkinter
+, wheel
+, wxPython_4_0
+}:
+
+buildPythonPackage rec {
+  pname = "deeplabcut";
+  version = "2.1.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b3c8c2a62d85c3fa188ecde04b473d38b9fba8e14537b4a88d4a82e61d0a1a2d";
+  };
+
+  resnet50 = fetchzip {
```

Comment:
I have a pending MR regarding this: https://github.com/NixOS/nixpkgs/pull/50679

Didn't finish it. But I believe the weights/models files should be coordinated together in Nixpkgs between all the ML packages.

Reply:
made new resnet pkg, good idea!